tools/add_triage_signature.py

```diff
@@ -1989,6 +1989,47 @@ def _process_ticket(self, url, issue_key):
         self._update_triaging_ticket(report)
 
 
+class HostsIPSignature(Signature):
+    def __init__(self, *args, **kwargs):
+        super().__init__(
+            *args,
+            **kwargs,
+            comment_identifying_string="h1. Host IPv4 addresses:",
+        )
+
+    def _process_ticket(self, url, issue_key):
+        md = get_metadata_json(url)
+
+        cluster = md["cluster"]
+
+        hosts = []
+        for host in cluster["hosts"]:
+            inventory = json.loads(host["inventory"])
+            hosts.append(
+                OrderedDict(
+                    id=host["id"],
+                    hostname=self._get_hostname(host),
+                    ipv4_addresses=self._get_interfaces(inventory)
```

Comment:
We should have one table row per interface, see "Host storage details" signature for how it does a similar thing. As of now ipv4_addresses is not readable enough